app/views/login/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class='login-main'>
+    <%= react_component("login_form",{path: log_path, token: @form_token, redirectPath: articles_path}) %>
```

Comment:
Looks like the route is still called `log_path`. Something to keep in mind in future when creating routes etc, if you weren't familiar with the work in this PR you would assume this was a path for logging. 

You're also missing a space after the comma ,

Reply:
Ive gone through and changed all instances of log_path to be more clear